shared/types.js

```diff
@@ -275,6 +275,8 @@ export type DBThread = {
   attachments?: Array<DBThreadAttachment>,
   edits?: Array<DBThreadEdits>,
   watercooler?: boolean,
+  messageCount: number,
+  reactionCount: number,
```

Comment:
This field will technically exist after this branch merges, but it's not being used - that's what i mean by my above comment